src/doc/rustc/src/command-line-arguments.md

```diff
@@ -345,6 +345,19 @@ replacement is purely textual, with no consideration of the current system's
 pathname syntax. For example `--remap-path-prefix foo=bar` will match
 `foo/lib.rs` but not `./foo/lib.rs`.
 
+<a id="option-remap-cwd-prefix"></a>
+## `--remap-cwd-prefix`: remap paths under the cwd in output
+
+Remap all absolute paths that are rooted under the current working directory to
+be under the given value instead. The given value may be absolute or relative,
+or empty. This switch takes precidence over `--remap-path-prefix` in case they
+would both match a given path.
+
+This flag allows the command line to be universally reproducible, such that the
+same execution will work on all machines, regardless of build environment.
+
+This is an unstable option. Use `-Z remap-cwd-prefix=val` to specify a value.
+
```

Comment:
As long as this option is unstable it needs to be documented under https://github.com/rust-lang/rust/tree/master/src/doc/unstable-book/src/compiler-flags rather than here.

Reply:
@danakj, would you mind moving this piece of documentation to the "unstable book" (as mentioned above)? Otherwise it would show up in the regular documentation which is only meant for stable features.

Reply:
Yes absolutely. I’ll be back at work next week and will do so then. Thanks very much!

Reply:
I've moved the documentation over.

I believe that https://github.com/rust-lang/rust/issues/87325 needs a tracking bug label attached to it, as well as an unstable label? But I am unable to do that. I thought of filing a new tracking but, but then I can't add the other labels it has.

Reply:
I went ahead and opened #89434 as a dedicated tracking issue.